### DISCUSSION TITLE
Question how can I create custom access token if I deployed supabase in self hosting docker ?

### DISCUSSION THREAD

**Discussion by Keliqq** [supabase/supabase#22957] [Questions]
I'm trying to create a Hook: Custom access token but I can't get it to work...
Supabase I deployed in self-hosting version.
The documentation says to add fields to config.toml I found it in supabase/supabase/config.toml . But as I understand it is not what I need.  Question how can I create custom access token if I deployed supabase in self hosting docker ? 

This is the documentation I used to create the Custom access token - 
https://supabase.com/docs/guides/auth/auth-hooks#hook-custom-access-token

This is the documentation I used to deploy supabase - 
https://supabase.com/docs/guides/self-hosting/docker

**Comment by Mox-I** [comment]
docker-compose.yml -> services.auth.environment -> 
GOTRUE_HOOK_CUSTOM_ACCESS_TOKEN_ENABLED
GOTRUE_HOOK_CUSTOM_ACCESS_TOKEN_URI

uncomment to enable custom access token hook.
